Content.Server.Database/Model.cs

```diff
@@ -100,6 +101,10 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
             modelBuilder.Entity<AdminLogPlayer>()
                 .HasKey(logPlayer => new {logPlayer.PlayerUserId, logPlayer.LogId, logPlayer.RoundId});
+
+            modelBuilder.Entity<Whitelist>()
+                .Property(w => w.UserId)
+                .IsRequired();
```

Comment:
couldn't you just use the attribute? additionally i dont think this is even needed, since keys are required implicitly iirc